src/oscar/apps/customer/abstract_models.py

```diff
@@ -117,7 +117,7 @@ class AbstractEmail(models.Model):
         related_name='emails',
         verbose_name=_("User"),
         null=True)
-    email = models.EmailField(_('Email Address'))
+    email = models.EmailField(_('Email Address'), null=True, blank=True)
```

Comment:
Yes but since this is a new field in a potentially very large table adding a new column with a default value ('' in this case) will lock the database.